planetarium/NineChronicles
# Fix Buff skill's stats to real buff stats

### Description

1. Fix Buff skill's stats to real buff stats

### How to test

1. Check table stats
2. Check crafted items

### Related Links

### Required Reviewers

@Namyujeong 

### Screenshot
![image](https://user-images.githubusercontent.com/60082897/133203688-1bd6ec55-c22b-4802-b726-3ca2556c1ee5.png)
![image](https://user-images.githubusercontent.com/60082897/133203701-9361e1d9-3e1b-4c2f-87cf-c5331f33fdfe.png)
